src/editor.js

```diff
@@ -44,16 +38,28 @@ CKEDITOR.define( [
 			 * global configurations available in {@link CKEDITOR.config} if configurations are not found in the
 			 * instance itself.
 			 *
-			 * @type {Config}
+			 * @readonly
+			 * @type {Config} config
 			 */
-			this.config = new EditorConfig( config );
+			this.set( 'config', new EditorConfig() );
 
 			/**
 			 * The plugins loaded and in use by this editor instance.
 			 *
-			 * @type {PluginCollection}
+			 * @readonly
+			 * @type {PluginCollection} plugins
 			 */
-			this.plugins = new PluginCollection( this );
+			this.set( 'plugins', new PluginCollection( this ) );
+
+			/**
+			 * The editables attached to this editor instance.
+			 *
+			 * @readonly
+			 * @type {EditableCollection} editables
+			 */
+			this.set( 'editables', new EditableCollection() );
+
+			this._creators = {};
```

Comment:
Creators are strictly related to editor instances:

 * They're loaded on demand through instance plugins. 
 * In most of the cases, an editor will load just one creator which would be the default one, making `config.creator` not necessary.
 * They could be customized (overridden) in a per instance fashion.

Creators have no use at `CKEDITOR`. Additionally we would have the problem of defining who should load them if not the editor instance.

Still one could argue that we could use `CKEDITOR` as the repository for creators but there is zero benefit on this. What we have in each instance is simply (1) a list of available creators for that specific editor instance with (2) pointers to the creator class. It's a very lightweight structure that fulfils the requirement of "1" giving quick access to the creators with "2".